Tasks/Common/ios-signing-common/ios-signing-common.ts

```diff
@@ -132,6 +132,54 @@ export async function findSigningIdentity(keychainPath: string) {
     }
 }
 
+    /**
+     * Get Cloud entitlement type Production or Development according to the export method - if entitlement doesn't exists in provisioning profile returns null 
+     * @param provisioningProfilePath
+     * @param exportMethod
+     * @returns {string} 
+     */
+export async function getCloudEntitlement(provisioningProfilePath: string, exportMethod: string): Promise<string> {
+    //find the provisioning profile details
```

Comment:
I noticed we already repeat this code in several places. It should be factored out into a function. But that is a pre-existing issue I can address in another PR, especially if you're planning to port this fix back to 2.0.